### PR TITLE
[JBIDE-22137] updated to openshift-restclient-java 4.0.3.Final

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Comment by adietish** [comment]
@fbricon @alexeykazakov please review

File: plugins/org.jboss.tools.openshift.client/build.properties [hunk 1]
```diff
@@ -13,5 +13,6 @@ bin.includes = META-INF/,\
                jboss_about.png,\
                .options,\
                lib/*,\
-               log4j.properties
+               log4j.properties,\
+               lib/openshift-restclient-java-4.0.3.Final.jar
```
Comment (fbricon) [inline plugins/org.jboss.tools.openshift.client/build.properties]: this change is not necessary

**Comment by alexeykazakov** [comment]
+1

**Comment by fbricon** [comment]
+1

**Comment by adietish** [comment]
merged into jbosstools-4.3.x and cherry-picked into master

**Pull request merged**